src/components/Footer/styles.scss

```diff
@@ -9,3 +9,20 @@
     height: rem(72px);
   }
 }
+@import '../../styles/main.scss';
+
+.footer {
+  height: rem(32px);
+  background-color: $secondary;
+  
+  @include flex-center();
+  @include respond-above(md) {
+     height: rem(72px);
+   }
+   
+  &__item {
+    font-size: 16px;
+    font-weight: 500;
+    line-height: normal;
+  }
+}
```

Comment:
According to the solution I've guided, there's no need for `.footer-container`. 🙅🏻‍♂️

Reply:
Seems like you've copy/paste the solution without any understandings. Please make sure you do understand what the code does. If you don't understand then raise a your hand without any hesitation and we'll help you. 🙇🏻‍♂️